app/bundles/LeadBundle/Command/SegmentStatCommand.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\LeadBundle\Command;
+
+use Mautic\CoreBundle\Command\ModeratedCommand;
+use Mautic\CoreBundle\Helper\CoreParametersHelper;
+use Mautic\CoreBundle\Helper\PathsHelper;
+use Mautic\LeadBundle\Event\GetStatDataEvent;
+use Mautic\LeadBundle\LeadEvents;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+class SegmentStatCommand extends ModeratedCommand
```

Comment:
Please make new classes `final` by default.
```suggestion
final class SegmentStatCommand extends ModeratedCommand
```